Mathlib/RingTheory/PowerSeries/PiTopology.lean

```diff
@@ -0,0 +1,207 @@
+/-
+Copyright (c) 2024 Antoine Chambert-Loir, María Inés de Frutos-Fernández. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Antoine Chambert-Loir, María Inés de Frutos-Fernández
+-/
+
+
+import Mathlib.RingTheory.MvPowerSeries.PiTopology
+import Mathlib.RingTheory.PowerSeries.Basic
+
+/-! # Product topology on power series
+
+Let `R` be with `Semiring R` and `TopologicalSpace R`
+In this file we define the topology on `PowerSeries σ R`
+that corresponds to the simple convergence on its coefficients.
+It is the coarsest topology for which all coefficients maps are continuous.
+
+When `R` has `UniformSpace R`, we define the corresponding uniform structure.
+
+This topology can be included by writing `open scoped PowerSeries.WithPiTopology`.
+
+When the type of coefficients has the discrete topology,
+it corresponds to the topology defined by [bourbaki1981], chapter 4, §4, n°2.
+
+It is *not* the adic topology in general.
```

Comment:
I'm surprised by this, I thought the topologies were the same in the one-variable case (assuming discrete topology on the coefficients). What's the counterexample?

Reply:
The comment is an unfortunate copy-paste from the multivariate case.